docusaurus.config.js

```diff
@@ -112,7 +117,7 @@ const config = {
             ],
           },*/
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} New Earth Art & Tech LLC. Built with Docusaurus.`,
```

Comment:
Lets remove the Build With Docusaurus

Reply:
leave the copyright, just remove "Build With Docusaurus", the line should say Copyright © ${new Date().getFullYear()} New Earth Art & Tech LLC.